hazelcast-windows-vm-cluster/azuredeploy.json

```diff
@@ -0,0 +1,292 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "adminUsername": {
+          "type": "String",
+          "metadata": {
+              "description": "Username for hazelcast VMs"
+          }
+      },
+      "adminPassword": {
+          "type": "SecureString",
+          "metadata": {
+              "description": "Password for hazelcast VMs"
+          }
+      },
+      "hazelcastVersion": {
+        "defaultValue": "3.11",
+        "type": "String",
+        "metadata": {
+            "description": "Hazelcast Version"
+        },
+        "allowedValues": [
+            "3.11",
+            "3.11.1"
+          ]
+    },
+      "clusterName": {
+          "defaultValue": "[parameters('adminUsername')]",
+          "type": "String",
+          "metadata": {
+              "description": "Username for hazelcast grid"
+          }
+      },
+      "dnsPrefix": {
```

Comment:
add a defaultValue and use uniqueString() to generate a unique one

Reply:
we generated unique value in `dnsSettings`